src/refactor_nrepl/ns/tracker.clj

```diff
@@ -37,8 +55,28 @@
           :when ((set deps) ns)]
       file)))
 
+(defn- in-refresh-dirs? [refresh-dirs file]
```

Comment:
Can add a unit test to this one. For now I didn't b/c my reasoning was that this is a private helper, and often private helpers are left untested.

Don't know how `clojure-emacs` folks tend to approach these - would be happy to tweak

Reply:
I don't typically add tests for private helpers.